app/api/users/router.py

```diff
@@ -0,0 +1,15 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+from typing import List
+from .service import get_all_user
+from .dependencies import get_db
+from app.api.user.schemas import User
```

Comment:
```suggestion
from typing import List

from fastapi import APIRouter, Depends
from sqlalchemy.orm import Session

from app.api.user.schemas import User
from .dependencies import get_db
from .service import get_all_user
```
<img width="789" alt="image" src="https://user-images.githubusercontent.com/49834964/204429265-4b98dfe6-aa16-4f22-aec3-fc89b9b04289.png">


https://peps.python.org/pep-0008/#imports